src/content/community/support/index.md

```diff
@@ -38,8 +38,11 @@ Looking for an Ethereum wallet? [Explore our full list of Ethereum wallets](/wal
 
 Building can be hard. Here are some development focused spaces with experienced Ethereum developers that are happy to help.
 
+- [Alchemy University](https://university.alchemy.com/#starter_code)
+- [Chainlink](https://blog.chain.link/learn-blockchain-full-stack-web3-javascript-smart-contract-development/)
```

Comment:
```suggestion
```

I'm not sure this belongs in this section of ethereum.org. We link out to this in the tutorials section, but my gut take is that a tutorial is not the same as an online community so this should be removed.

Reply:
Hey Corwin - gotcha thanks, that makes sense, I'll defer to you! I didn't want to just link Alchemy docs but more so, resources that would benefits everyone. Just for context FWIW, we just see a ton of traffic and developers mentioning Patrick Collins specifically (Chainlink devrel) who is a force (in an awesome way) for the web3 dev space - he does a infamous 32 hour web3 dev Youtube video haha, that's the video on this Chainlink page.